src/shortcodes/image-and-text.js

```diff
@@ -1,8 +1,9 @@
 const markdownFilter = require("../filters/markdown.js");
 
-module.exports = (content, image, alt, imagePosition, verticalAlignment) => {
+module.exports = (content, image, alt, title, imagePosition, verticalAlignment) => {
+	const imageCaption = title ? `title="${title}"` : "";
```

Comment:
Instead of doing this, the caption should be processed with Markdown and inserted directly into the markup.